shared/src/validation-schemas/user/user.ts

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+import 'yup-phone';
+import { RegisterPayloadKey, UserType, UserSex, RegisterValidationRule, RegisterValidationMessage } from '../../common/enums'
```

Comment:
```suggestion
import { RegisterPayloadKey, UserType, UserSex, RegisterValidationRule, RegisterValidationMessage } from '~/common/enums'
```
use the alias in the shared folder